templates/package.xml.template

```diff
@@ -31,15 +31,18 @@
       <file baseinstalldir="/" name="config.m4" role="src" />
       <file baseinstalldir="/" name="config.w32" role="src" />
       <file baseinstalldir="/" name="src/php/README.md" role="src" />
-      % for source in php_config_m4.src + php_config_m4.headers:
-      <file baseinstalldir="/" name="${source}" role="src" />
-      % endfor
-      % for lib in libs:
-      % if lib.name in php_config_m4.get('deps', []):
-      % for source in lib.get('public_headers', []) + lib.headers + lib.src:
-      <file baseinstalldir="/" name="${source}" role="src" />
-      % endfor
-      % endif
+  <%
```

Comment:
Somehow it introduces a newline somewhere here as mentioned above.

Reply:
Yeah, I removed it.